docs/ops/sort/MatrixNMS_8.md

```diff
@@ -0,0 +1,157 @@
+  * **Required**: *No*
+
+* *keep_top_k*
+
+  * **Description**: maximum number of boxes to be selected per batch element.
+  * **Range of values**: an integer starting from `-1`.
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *background_class*
+
+  * **Description**: the background class id.
+  * **Range of values**: an integer from `-1` to the maximum class id
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all classes
+  * **Required**: *No*
+
+* *decay_function*
+
+  * **Description**: decay function used to decay scores.
+  * **Range of values**: `gaussian`, `linear`
+  * **Type**: `string`
+  * **Default value**: `linear`
+  * **Required**: *No*
+
+* *gaussian_sigma*
+
+  * **Description**: gaussian_sigma parameter for gaussian decay_function.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `2.0`
+  * **Required**: *No*
+
+* *post_threshold*
+
+  * **Description**: threshold to filter out boxes with low confidence score `after` decaying.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. he box cooridnates are layout as `[xmin, ymin, xmax, ymax]`. Required.
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_batches, num_classes, num_boxes]` with box scores. Required.
+
+**Outputs**:
+
+*   **1**: `selected_outputs` - tensor of type *T_THRESHOLDS* and shape `[number of selected boxes, 6]` containing the selected boxes with score and class as triplets `[class_id, box_score, xmin, ymin, xmax, ymax]`.
+
+*   **2**: `selected_indices` - tensor of type *T_IND* and shape `[number of selected boxes, ]` the selected indices in the flattened `boxes`, which are absolute values cross batches.
+
+*   **3**: `selected_num` - 1D tensor of type *T_IND* and shape `[num_batches, ]` representing the number of selected boxes for each batch element.
+
+When there is no box selected, `selected_num` will be filled with 0. And both `selected_outputs` and `selected_indices` are null of shape (0,).
+
+**Types**
+
+* *T*: floating point type.
+
+* *T_MAX_BOXES*: integer type.
+
+* *T_THRESHOLDS*: floating point type.
+
+* *T_IND*: `int64` or `int32`.
+
+**Example**
+
+```xml
+<layer ... type="MatrixNms" ... >
+    <data decay_function='guassian' sort_result="score" output_type="i64"/>
+    <input>
+        <port id="0">
+            <dim>3</dim>
+            <dim>100</dim>
+            <dim>4</dim>
+        </port>
+        <port id="1">
+            <dim>3</dim>
+            <dim>5</dim>
+            <dim>100</dim>
+        </port>
+        <port id="2"/>
+        <port id="3"/>
+        <port id="4"/>
+    </input>
+    <output>
+        <port id="5" precision="FP32">
+            <dim>-1</dim>
```

Comment:
Add a comment here that "-1" means a undefined dimension calculated during the model inference.